docs/pages/9 - Contrib Modules.md

```diff
@@ -165,76 +165,68 @@ object app extends ScalaModule with TwirlModule {
 } 
 ``` 
 
+### Play Framework
+
+Play framework routes generation support.
+
+ 
+To declare a module that needs to generate Play Framework routes , you must mix-in the 
```

Comment:
```suggestion
To declare a module that needs to generate Play Framework routes, you must mix-in the 
```

Reply:
(i'll fixup the fixes and push once I have resolved all comments if that's acceptable. I'll mark :+1: on all comments I have processed)